helm/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: squid-exporter
+description: A Helm chart for Kubernetes
```

Comment:
nit: Please add a more meaningful description here.

Reply:
Thank you, done.